opm/simulators/flow/CpGridVanguard.hpp

```diff
@@ -269,6 +272,13 @@ class CpGridVanguard : public FlowBaseVanguard<TypeTag>
         return this->grid().globalCell();
     }
 
+    /*!
+     * \brief Returns the object which maps a global element index of the simulation grid
+     *        to the corresponding element index of the level logically Cartesian index.
+     */
+    const LevelCartesianIndexMapper levelCartesianIndexMapper() const
```

Comment:
Is it correct that this function is here and not in GenericCpGridVanguard.hpp? If so, then leave it as it is.

Reply:
Oh no! Thanks for catching that. It should be removed.